charts/envoy/values.yaml

```diff
@@ -124,6 +124,29 @@ tls:
     certChainSecret: ""
     # -- Name of the Secret containing the private key file used for TLS communication.
     privateKeySecret: ""
+    certManager:
+      # -- Use cert-manager to manage private key and certificate files.
+      enabled: false
+      # -- Duration of a certificate.
+      duration: "87600h0m0s"
```

Comment:
Isn't it 3650 days? Should it be `8760h`?

Reply:
Ah, yes. You are right.
According to the previous offline discussion, I will update this default value to `8760h (365 days)`.

Reply:
Fixed in 26dc8459235933e38de6bcdae4e84c1909d9b1ce.